admin/v1/admin.proto

```diff
@@ -583,3 +590,18 @@ message ExportAuditLogsRequest {
   // Required. The UTC date to end data from. If not set, the first day of the current year will be used. Format: `yyyymmdd`.
   string endTime = 3;
 }
+
+// for wave features settings data maping
+message WaveFeature {
+  string key = 1;
+  string value = 2;
```

Comment:
Please change `value` to boolean. This is either enabled or disabled only, right?